server/src/test/java/org/cloudfoundry/identity/uaa/impl/config/AwsSecretsLoaderTest.java

```diff
@@ -71,7 +72,7 @@ void cleanup() {
         System.clearProperty("AWS_SECRET_MANAGER_NAMES");
     }
 
-    String vaidYamlString() {
+    String validYamlString() {
```

Comment:
No need to have it as a method. Convert to constant

Reply:
Fixed by Irina